docs/management_api.yml

```diff
@@ -484,53 +484,83 @@ paths:
             application/json:
               - name: my-app-v1.0.1
                 artifacts:
-                  - name: my-app-v1.0.1
-                    description: Application v1.0.1
+                  - id: 0c13a0e6-6b63-475d-8260-ee42a590e8ff
+                    name: Application 1.0.0
+                    description: Johns Monday test build
                     device_types_compatible: [Beagle Bone]
-                    id: 0c13a0e6-6b63-475d-8260-ee42a590e8ff
-                    signed: false
-                    modified: "2016-03-11T13:03:17.063493443Z"
                     info:
                       type_info:
                         type: rootfs
-                    files:
-                      - name: rootfs-image-1
-                        checksum: cc436f982bc60a8255fe1926a450db5f195a19ad
-                        size: 23421351
-                        date: 2016-03-11T13:03:17.063+0000
-                    metadata: {}
-                  - name: my-app-v1.0.1
-                    description: Application v1.0.1
-                    device_types_compatible: [Raspberry Pi]
-                    id: 0c13a0e6-6b63-475d-8260-ee42a590e8ff
                     signed: false
+                    updates:
+                      type_info: "rootfs"
```

Comment:
Aren't these more commonly "rootfs-image"?

Reply:
Yes, you are right; copy-and-past betrayed me.